tools/release-plugin.sh

```diff
@@ -261,6 +260,7 @@ function do_packagist_check {
 	# We expect a new version when (1) the package is touched in this release and (2) it has no change entry files remaining.
 	POLL_ARGS=()
 	cd "$BASE"
+	GITBASE=$(git merge-base trunk HEAD)
```

Comment:
This is probably ok, but might be slightly more robust as
```suggestion
	GITBASE=$(git merge-base origin/trunk HEAD)
```
in case someone did a `git fetch` and such without updating the local copy of the trunk branch.